ui/app/AppLayouts/Onboarding2/pages/SeedphrasePage.qml

```diff
@@ -0,0 +1,67 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+
+import shared.panels 1.0
+
+OnboardingPage {
+    id: root
+
+    title: qsTr("Create profile using a recovery phrase")
+    property string subtitle: qsTr("Enter your 12, 18 or 24 word recovery phrase")
+    property alias btnContinueText: btnContinue.text
+
+    property var isSeedPhraseValid: (mnemonic) => { console.error("isSeedPhraseValid IMPLEMENT ME"); return false }
```

Comment:
Does this need a Nim function? If so, you can probably just call the store for now